sig-network/role-sig-chair.md

```diff
@@ -0,0 +1,70 @@
+# Role: SIG Chair
+
+## Background
+
+Please see the
+[standard description of a SIG chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair),
+which does a good job describing the expectations of the more “administrative”
+aspect of being a SIG chair.
+
+## What's expected
+
+Rather than repeat the doc linked above, this doc will just add some color.
+
+The minimum effective engagement for a SIG chair is to “run the show” (really
+“make sure the show is running” - delegation is good!).  That generally means
+building an agenda for and running regular meetings, curating our
+community-facing metadata, and acting as liaison to other groups (other SIGs
+and steering, mostly).
+
+This includes new-issue triage - either doing the triage or leading it (e.g. in
+the regular SIG meetings).
+
+## Above and beyond
+
+The basic expectations above should not be a huge time commitment.  SIG chairs
+who want to have more impact and can commit more time can engage in a multitude
+of additional ways.  While this can include being an active code contributor or
+SIG tech-lead, an effective SIG chair does not **need** to be a code
+contributor.
+
+Some concrete things that SIGs need to do, which the chair-people can
+facilitate, participate in, or even do themselves include:
+  * Active new-issue triage - go through new issues, ask for more info if
+    needed, apply labels, close or redirect “support” requests, CC people who
+    might have context, de-dup, triage-accept obvious bugs, ping submitters for
+    updates, etc.
+  * Backlog grooming - go through older issues, re-evaluate triage, de-dup,
+    aggregate similar issues, cross-link issues, link to KEPs, probe for
+    updates, etc.
+  * Manage issue assignments - go through issues which are assigned to people
+    and see if progress is being made or if assignments are stale.
+  * Curate "help wanted" and "good first issue" issues - file new issues or
+    improve existing issues (e.g. by adding details or getting others to do so)
+    to enable new contributors to take them up.
+  * Manage project board(s) to organize information - things like KEPs, issues,
+    PRs, or in-progress dev efforts can usually benefit from organizing and
+    curation.
+  * Manage KEPs - track the KEP lifecycle, update metadata, ping KEP owners for
```

Comment:
this is a requirement from the [document](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair:~:text=SHOULD%20identify%2C%20track%2C%20and%20maintain%20the%20SIGs%20enhancements%20for%20current%20release%20and%20serve%20as%20point%20of%20contact%20for%20the%20release%20team%2C%20but%20MAY%20delegate%20to%20another%20Lead%20to%20fulfill%20these%20responsibilities):

> SHOULD identify, track, and maintain the SIGs enhancements for current release and serve as point of contact for the release team, but MAY delegate to another Lead to fulfill these responsibilities

Does this refers to the fact that to go beyond one should not delegate? Or that tracking may be more involved than just checking the status of PRs in enhancements repo?

Reply:
At least in sig-net, much of the KEP tracking is done as a group.  I think the basic requirement is pinging people, but there's a lot more that CAN be done.  Will update a bit.